content/blog/2020-07-31-spreadsheet-workflows-using-r/index.Rmarkdown

```diff
@@ -0,0 +1,185 @@
+---
+title: Spreadsheets Workflows in R
+date: '2020-07-31'
+slug: spreadsheets-using-r
+categories:
+  - learn
+tags:
+  - spreadsheets
+  - tidyverse
+author:
+  - Devin Johnson
+photo:
+  url: https://unsplash.com/photos/Wpnoqo2plFA
+  author: Mika Baumeister
+---
+
+There's so much to discover and learn when working with spreadsheets, and creating a resource that speaks to both spreadsheet and R users has been an exciting challenge.  While my internship has focused on a plethora of topics from comparing numerous R package functionalities that work with spreafsheets, to writing tips and tricks on best practices for working in both spreadsheets and RStudio, what I'd like to focus on in this post is a section dedicated to the R versions of common spreadsheet workflows.
```

Comment:
Feels like this should kick off with a very plain intro of yourself as an RStudio intern, creating resources for people and project at the interface between spreadsheets and R.

As it stands, that is _implicit_, but you sort of have to back this out.

So the paragraph content feels right, but somewhat in the wrong order.

Reply:
I think I get what you mean here so have a revision of this intro paragraph that touches on that more explicitly.